BrowserKit/Sources/ComponentLibrary/Buttons/PrimaryRoundedButton.swift

```diff
@@ -59,7 +60,7 @@ public class PrimaryRoundedButton: LegacyResizableButton, ThemeApplicable {
         highlightedTintColor = theme.colors.actionPrimaryHover
         normalTintColor = theme.colors.actionPrimary
 
-        setTitleColor(theme.colors.textInverted, for: .normal)
-        backgroundColor = theme.colors.actionPrimary
+        configuration?.baseForegroundColor = theme.colors.textInverted
```

Comment:
With these change the button will not use the highlighted color anymore. Could you please take a look at [this PR](https://github.com/mozilla-mobile/firefox-ios/pull/17597) and change the highlighted state accordingly?

Reply:
Done 👍